### DISCUSSION TITLE
Would it be possible to implement the old Chromium UI?

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1418] [Ideas]
Perhaps through adding back the [flags](https://www.ytechb.com/wp-content/uploads/2018/09/How-to-get-old-chrome-ui-back-4-696x391.png) which used to enable it? This is pretty much the only reason I do not use it over hardened Firefox. I hope this is not a silly idea...

**Comment by TobyTobyReal** [comment]
Bump! I really wish it was possible :(

**Reply by networkException** [reply]
It is not

**Reply by TobyTobyReal** [reply]
womp

**Reply by PF4Public** [reply]
It _is_ possible, the thing is that it is unfortunately _impractical_
